src/components/dropdownSearch/DropdownSearchReducers.tsx

```diff
@@ -111,11 +154,17 @@ export const dropdownSearchReducer = (state: IDropdownSearchState = dropdownSear
         return {
           ...state,
           id: action.payload.id,
-          selectedOption: state.activeOption,
           isOpened: false,
+          selectedOptions: state.selectedOptions.push(state.activeOption),
```

Comment:
You're right! but FixedOption.push returns a new object! So the previous state is not being changed